# two points in spherical coordinates: left and right.
# these two points represent the extreme visible ends of
# the bar. Spherical coordinates means that they're being
# represented in horizon and vertical radians, where '0'
# is looking straight ahead from the camera (x axis).
# distance to the point is measured in mm.

float32 l_horizon_rads
float32 l_vertical_rads
float32 l_dist_mm

float32 r_horizon_rads
float32 r_vertical_rads
float32 r_dist_mm
